examples/Sql/HiveDataFrame/Program.cs

```diff
@@ -24,7 +24,10 @@ static void Main(string[] args)
             var sparkConf = new SparkConf();
             var sparkContext = new SparkContext(sparkConf);
             var hiveContext = new HiveContext(sparkContext);
-            var peopleDataFrame = hiveContext.Read().Json(Path.Combine(Environment.CurrentDirectory, @"data\people.json"));
```

Comment:
Would specifying file in format "file:///C:/path/to/inputfile" work? If you so, you may want to specify that in the instructions to run examples (similiar to https://github.com/Microsoft/Mobius/blob/master/notes/running-mobius-app.md#json-example---f-sql)